### ISSUE TITLE
vault_aws_secret_backend_role fails creating role with policy_arn

### ISSUE THREAD

**Issue by robertlabrie** [terraform-providers/terraform-provider-vault#82]
Using vault to manage non-secret assets in vault. I can create a vault_aws_secret_backend_role using `policy` but it throws a 500 using `policy_arn`.

### Terraform Version
```
Terraform v0.11.3
+ provider.vault v1.0.0

also vault 0.9.3
```
### Affected Resource(s)
- vault_aws_secret_backend_role
### Terraform Configuration Files
```hcl
provider "vault" {
  address = "https://vault.example.com"
}

terraform {
  backend "s3" {
    bucket = "example-terraform-statefiles"
    key    = "us-east-1/vault-policy.tfstate"
    region = "us-east-1"
    lock_table = "terraform_statelock"
  }
}
```

### Debug Output
These logs are full of tokens and other stuff I'm not going to redact. For this resource:
```hcl
resource "vault_aws_secret_backend_role" "vault_node_policy" {
  backend = "aws"
  name    = "vault-node-policy"
  policy_arn = "arn:aws:iam::1234567890000:policy/vault_node_policy"
}
```

I get this error:
```
* vault_aws_secret_backend_role.vault_node_policy: Error creating role "vault-node-policy" for backend "aws": Error making API request.

URL: PUT https://vault.ops.tropo.com/v1/aws/roles/vault-node-policy
Code: 500. Errors:
```

### Expected Behavior
Role was created in vault backend

### Actual Behavior
Got a 500 error from vault

### Steps to Reproduce
Please list the steps required to reproduce the issue, for example:
1. `terraform apply`

**Comment by robertlabrie** [comment]
And it was literally fixed here: https://github.com/terraform-providers/terraform-provider-vault/commit/c123a5b04a6c94a31472c58c46b8390233b348a9

When do we get a release?

**Comment by martinssipenko** [comment]
This is a duplicate of #60 which was fixed by #49.

**Comment by catsby** [comment]
Closing, was added/fixed in https://github.com/terraform-providers/terraform-provider-vault/pull/49
